js/indicators/ichimoku-kinko-hyo.src.js

```diff
@@ -252,26 +280,51 @@ seriesType('ikh', 'sma',
                 /**
                  * Color of the line.
                  *
-                 * @type {Number}
+                 * @type {Highcharts.ColorString}
                  * @since 6.0.0
                  * @product highstock
                  */
                 lineColor: undefined
             }
         },
         /**
-         * The styles for fill between Senkou Span A and B
+         * The styles for area between Senkou Span A and B
          *
          * @since 6.0.0
          * @product highstock
          */
         senkouSpan: {
+            /**
+            * Color of the area between Senkou Span A and B,
+            * when Senkou Span A is above Senkou Span B. Note that if
+            * a `style.fill` is defined, the `color` takes precedence and
+            * the `style.fill` is ignored.
+            *
+            * @type {Highcharts.ColorString}
+            * @since next
+            * @product highstock
+            * @apioption plotOptions.ikh.senkouSpan.color
+            * @default undefined
+            * @see [senkouSpan.styles.fill](#series.ikh.senkouSpan.styles.fill)
+            */
+
+            /**
+            * Color of the area between Senkou Span A and B,
+            * when Senkou Span A is under Senkou Span B.
+            *
+            * @type {Highcharts.ColorString}
+            * @since next
+            * @product highstock
+            * @apioption plotOptions.ikh.senkouSpan.negativeColor
```

Comment:
Please add an `@sample`. The same sample can be used for `senkouSpan.color`, just link it.